src/task/MonitorTask.ts

```diff
@@ -283,14 +293,11 @@ export default class MonitorTask
       }
```

Comment:
Given we already have the build code in the transition guide, I'm fine moving this logging to the demo too? that way its a more usable example.